src/vs/workbench/electron-browser/commands.ts

```diff
@@ -373,15 +376,31 @@ export function registerCommands(): void {
 		win: { primary: void 0 }
 	});
 
-	CommandsRegistry.registerCommand('_workbench.diff', function (accessor: ServicesAccessor, args: [URI, URI, string, string]) {
+	CommandsRegistry.registerCommand('_workbench.diff', function (accessor: ServicesAccessor, args: [URI, URI, string, string, vscode.TextDocumentShowOptions]) {
```

Comment:
I don't see an `IEditorOption` -- there is `IEditorOptions`, but that interface doesn't have any of the fields. Are you suggesting adding them there? Or am I missing something?